src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseHybridFVM.cpp

```diff
@@ -47,6 +47,7 @@ SinglePhaseHybridFVM::SinglePhaseHybridFVM( const string & name,
   m_areaRelTol( 1e-8 )
 {
 
```

Comment:
forgotten?

Reply:
I think you should check another time all the `SolverBase` children logging calls.